frontend/src/pages/org/dashboard.ts

```diff
@@ -0,0 +1,260 @@
+import type { PropertyValues, TemplateResult } from "lit";
+import { state, property } from "lit/decorators.js";
+import { when } from "lit/directives/when.js";
+import { msg, localized, str } from "@lit/localize";
+import type { SlSelectEvent } from "@shoelace-style/shoelace";
+
+import LiteElement, { html } from "../../utils/LiteElement";
+import type { AuthState } from "../../utils/AuthService";
+import type { OrgData } from "../../utils/orgs";
+import type { SelectNewDialogEvent } from "./index";
+
+type Metrics = {
+  storageUsedBytes: number;
+  storageUsedGB: number;
+  storageQuotaBytes: number;
+  storageQuotaGB: number;
+  archivedItemCount: number;
+  crawlCount: number;
+  uploadCount: number;
+  pageCount: number;
+  profileCount: number;
+  workflowsRunningCount: number;
+  maxConcurrentCrawls: number;
+  workflowsQueuedCount: number;
+  collectionsCount: number;
+  publicCollectionsCount: number;
+};
+
+@localized()
+export class Dashboard extends LiteElement {
+  @property({ type: Object })
+  authState!: AuthState;
+
+  @property({ type: String })
+  orgId!: string;
+
+  @property({ type: Object })
+  org: OrgData | null = null;
+
+  @state()
+  private metrics?: Metrics;
+
+  willUpdate(changedProperties: PropertyValues<this>) {
+    if (changedProperties.has("orgId")) {
+      this.fetchMetrics();
+    }
+  }
+
+  render() {
+    return html`<header class="flex justify-between gap-2 pb-3 mb-7 border-b">
```

Comment:
Move org settings from nav menu to here if admin still?  Maybe we decide this along with #1202 or address it later?  There's another place it could go in a new nav.

Reply:
Oversight on my part not adding the shortcut to settings, will do. IMO we should keep the "Org Settings" navigation item or have it written out as "settings" somewhere else, it wasn't clear to me from the mockups that the gear should go to org settings--discuss in Discord?